core/src/components/alert/alert.tsx

```diff
@@ -381,22 +381,39 @@ export class Alert implements ComponentInterface, OverlayInterface {
     }
     return (
       <div class="alert-input-group" aria-labelledby={labelledby}>
-        { inputs.map(i => (
-          <div class="alert-input-wrapper">
-            <input
-              placeholder={i.placeholder}
-              value={i.value}
-              type={i.type}
-              min={i.min}
-              max={i.max}
-              onInput={e => i.value = (e.target as any).value}
-              id={i.id}
-              disabled={i.disabled}
-              tabIndex={0}
-              class="alert-input"
-            />
-          </div>
-        ))}
+        { inputs.map(i => {
+          if (i.type === 'textarea') {
+            return (<div class="alert-input-wrapper">
+                      <textarea
```

Comment:
Can you add
```
/// @prop - Minimum height of a textarea in the alert
$alert-input-min-height:          37px !default;
```
to `alert.vars.scss`

and 

```
textarea.alert-input {
  min-height: $alert-input-min-height;
  
  resize: vertical;
}
```
to `alert.scss`

I would push the changes myself, but I do not have access to push to your branch.